projects/cds/src/profiletag/profile-tag.model.ts

```diff
@@ -0,0 +1,21 @@
+export interface ProfileTagWindowObject {
+  Y_TRACKING: {
+    push?: Function;
+    q?: ProfileTagJsConfig[][];
+  };
+}
+
+export interface ProfileTagJsConfig {
+  tenant?: string;
+  siteId?: string;
```

Comment:
Since this object is used only for you js script, it's fine if it has required properties. Only the customer-facing config objects should have all properties optional. It's up to you if you want to change it.